src/style/_fonts.scss

```diff
@@ -0,0 +1,34 @@
+@font-face{
+    font-family: 'Roboto-Light';
+    font-display: swap;
+    src: url('./fonts/roboto-light-webfont.woff2') format('woff2'),
+    url('./fonts/roboto-light-webfont.woff') format('woff')
+}
+
+@font-face{
+    font-family: 'Roboto-Regular';
+    font-display: swap;
+    src: url('./fonts/roboto-regular-webfont.woff2') format('woff2'),
+    url('./fonts/roboto-regular-webfont.woff') format('woff')
+}
+
+@font-face{
+    font-family: 'Roboto-Medium';
+    font-display: swap;
+    src: url('./fonts/roboto-medium-webfont.woff2') format('woff2'),
+    url('./fonts/roboto-medium-webfont.woff') format('woff')
+}
+
+@font-face{
+    font-family: 'Roboto-Bold';
+    font-display: swap;
+    src: url('./fonts/roboto-bold-webfont.woff2') format('woff2'),
```

Comment:
**Change request**: can you set the output directory for the font assets to be `dist/fonts/`? 

While keeping `dist` flat is an admirable goal, the font assets clutter the folder 😄

Reply:
done